static/files/forms-data.json

```
@@ -581,6 +581,67 @@
           ]
         }
       ]
+    },
+    "/solutions/open-source-security/cyber-resilience-act": {
+      "templatePath": "/solutions/open-source-security/cyber-resilience-act/index.html",
+      "isModal": true,
+      "modalId": "contact-modal",
+      "formData": {
+        "title": "Get in touch with our experts on the Cyber Resilience Act (CRA) compliance",
+        "formId": "6051",
+        "returnUrl": "/solutions/open-source-security/cyber-resilience-act/index.html#contact-form-success",
+        "product": ""
+      },
+      "fieldsets": [
+        {
+          "title": "Fill in this form and a member of our team will be in touch shortly.",
+          "id": "contact-form",
+          "fields": [
+            {
+              "type": "text",
+              "label": "First name",
+              "id": "firstName",
+              "placeholder": "",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "label": "Last name",
+              "id": "lastName",
+              "placeholder": "",
+              "isRequired": true
+            },
+            {
+              "type": "email",
+              "id": "email",
+              "label": "Email address",
+              "placeholder": "",
+              "isRequired": true
+            },
+            {
+              "type": "tel",
+              "id": "telephone",
+              "label":"Mobile/cell phone number",
+              "placeholder": "",
```

This can be removed as it is not used
```suggestion
              "placeholder": "",
```